server/routes.js

```diff
@@ -249,6 +249,7 @@ function getWithConfig(app, route, handler, opts = {}) {
  * @param {boolean} opts.mobileApiEnabled If set to true, then *&#47;mobile-data.json* will respond to mobile API requests. This is primarily used by the React Native starter kit. (default: true)
  * @param {Array<string>} opts.mobileConfigFields List of fields that are needed in the config field of the *&#47;mobile-data.json* API. This is primarily used by the React Native starter kit. (default: [])
  * @param {boolean} opts.templateOptions If set to true, then *&#47;template-options.json* will return a list of available components so that components can be sorted in the CMS. This reads data from *config/template-options.yml*. See [Adding a homepage component](https://developers.quintype.com/malibu/tutorial/adding-a-homepage-component) for more details
+ * @param {boolean} opts.ampStoryPages If set to true, then all story pages will render amp pages.
```

Comment:
rename to something like canonicalAmpStoryPages or something to point out that the amp version of the story is the canonical version